docs/source/getting_started/overview.rst

```diff
@@ -3,28 +3,36 @@
 Overview
 --------
 
-*JupyterLab is the next generation user interface for Project Jupyter*
+*JupyterLab is the next-generation user interface for Project Jupyter.
+JupyterLab will eventually replace the classic Jupyter Notebook.*
 
-JupyterLab goes beyond the classic Jupyter Notebook by providing a
-flexible and :ref:`extensible <user_extensions>` web application with a set of reusable
-components. You can arrange multiple notebooks, text editors, terminals,
-output areas, and custom components using tabs/panels and collapsible
-sidebars. These :ref:`components <interface>` are carefully designed to enable you to use
-them together or on their own to support new workflows (for example, you
-can send code from a file to a :ref:`code console <code_console>` with a keystroke, or move
-cells around a :ref:`notebook <notebook>` or between notebooks with drag-and-drop).
+JupyterLab is an extensible open-source web-based interface for Project Jupyter.
+In JupyterLab, you can arrange multiple notebooks, text editors, terminals,
+output areas, and custom components, using tabs, adjustable splitters, and
+collapsible sidebars. JupyterLab extensions can customize or enhance any part of
+the JupyterLab, including providing custom components, themes, and new file
+viewers and editors.
 
-:ref:`JupyterLab has full support for Jupyter Notebook documents. <notebook>` In
-addition, it enables other models of interactive computing, such as:
+:ref:`JupyterLab has full support for Jupyter Notebook documents. <notebook>`
+JupyterLab also enables other models of interactive computing:
 
 -  :ref:`code_console` provide transient scratchpads for running code
-   interactively, with full support for rich output.
--  :ref:`Kernel-backed documents <kernel-backed-documents>` allow code in any text file (Markdown,
-   Python, R, LaTeX, etc.) to be run interactively in any Jupyter
-   kernel.
+   interactively, with full support for rich output. A code console can be
+   linked to a notebook kernel as a computation log from the notebook, for
+   example.
+-  :ref:`Kernel-backed documents <kernel-backed-documents>` enable code in any
+   text file (Markdown, Python, R, LaTeX, etc.) to be run interactively in any
+   Jupyter kernel.
+-  Notebook cell outputs can be pulled into their own tab, side-by-side with
+   the notebook, enabling simple dashboards with interactive controls backed by
+   a kernel.
+-  Multiple views of documents with different editors or viewers enable live
+   editing of documents reflected in other viewers. For example, it is easy to
+   have live preview of Markdown documents, or edit GeoJSON files with live
+   updates of an adjoining map.
 
 JupyterLab also offers a unified model for viewing and handling data
-formats. This allows data in many formats (images, CSV, JSON, Markdown,
+formats. This enables data in many formats (images, CSV, JSON, Markdown,
```

Comment:
In this case, I disagree with the use of "enable" since "enable" can be used without another verb.  As a result "to be opened" comes as a bit of a surprise to the reader.  I'd rearrange this sentence if we have to use enable.

Reply:
Good point. I'll take a crack at it, but would love suggestions!